### DISCUSSION TITLE
Can 19c EE VM (12 vCPU & 80 GB RAM) be replaced with Elcarro Oracle Operator?

### DISCUSSION THREAD

**Discussion by Fady-Ibra** [GoogleCloudPlatform/elcarro-oracle-operator#343] [Q&A] [answered]
Hello Wonderful Community,

We have an Oracle 19c Enterprise Edition VM with 12 vCPU, 80 GB Memory, and 1TB Hard Disk.
My questions:
- Is it suitable to use Eelcarro Oracle Operator instead of the VM, or is there any limitation for the CPU and Memory?
- Can Data Guard (or any alternative service) be used to sync the data with an on-prem Oracle database?

Kind regards,
Fady

**Comment by tabbyl21** [comment, marked as answer]
Hi Fady,

Thank you for your questions! 

El Carro is suitable for use instead of a VM. You can initially set the CPU and Memory you want to allocate to the database. Throughout an instance's lifecycle you are able to resize and increase the CPU and Memory used. 

Regarding your second question yes you can, here are instructions on how to do so: https://github.com/GoogleCloudPlatform/elcarro-oracle-operator/blob/main/docs/content/data-migration/automated-dataguard.md . 

Best,
Tabatha

**Reply by Fady-Ibra** [reply]
Hi Tabatha,

Thank you for your reply and great support by providing this link.

I have another question: 

- **Is using a GKE Standard cluster mandatory, or can a GKE Auto Cluster be used?**

I appreciate your continued support.

Kind regards,
Fady

**Reply by tabbyl21** [reply]
Hi Fady,

Currently El Carro is optimized to run on GKE Standard clusters.

Do you have a use case for GKE Auto Cluster? If so I would love to hear more about it and how we can support it. 

Best,
Tabatha

**Reply by Fady-Ibra** [reply]
Hi Tabatha,

I hope you are doing fine. 
Thank you for your continuous support. I appreciate it.

My concern is about the cost. And how to implement Oracle database with the least possible price.
I will work on the GKE Auto Cluster and update you.

Again, thank you for continuing to reply to me.

Kind regards,
Fady
